weaver/Dockerfile.build

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7-alpine
+FROM golang:1.10-alpine
```

Comment:
:thinking:

Reply:
no 1.11 images have been pushed to docker hub

Reply:
:thinking: